ckanext/vocabulary_services/cli.py

```diff
@@ -130,5 +131,42 @@ def init_db_cmd():
     click.secho(u"Vocabulary services tables are setup", fg=u"green")
 
 
+@click.command(u"vocabulary-services-update-db")
+def update_db_cmd():
```

Comment:
I like how you implemented this method but I wonder if it can be done using the migration script the CKAN uses. 
Check out the docos:
https://docs.ckan.org/en/latest/contributing/database-migrations.html#db-migrations
https://docs.ckan.org/en/latest/extensions/best-practices.html?highlight=migration#use-migrations-when-introducing-new-models

We have used it before, check out the other migrations scripts `ckanext-vocabulary-services/ckanext/vocabulary_services/migration/versions`
I am just not sure if we can use it to update a table with default null values.

Reply:
implemented the migration

Reply:
Can we remove this now or do you think we should keep this cli command around?

Reply:
removed